### PR TITLE
Vet clinic database: add "join table" for visits

### PR DESCRIPTION
**Pull request by MohamedHNoor**
- [x] Created a table named vets, specializations, and visits
- [x] Inserted data into these tables
- [x]  Wrote queries 

### Screenshot of the results of  queries:

Screenshot 2023-02-22 at 22 07 43

Screenshot 2023-02-22 at 22 08 30

Screenshot 2023-02-22 at 22 09 23

Screenshot 2023-02-22 at 22 09 41

Screenshot 2023-02-22 at 22 11 15

### CODE REVIEW

File: data.sql [hunk 1]
```diff
@@ -251,4 +251,81 @@ WHERE name IN ('Charmander', 'Squirtle', 'Blossom');
...
+  animal_id,
+  vet_id,
+  date_of_visit
+) VALUES 
+    ((SELECT id FROM animals WHERE name = 'Agumon'), (SELECT id FROM vets WHERE name = 'William Tatcher'), '2020-5-24'),
+    ((SELECT id FROM animals WHERE name = 'Agumon'), (SELECT id FROM vets WHERE name = 'Stephanie Mendez'), '2020-7-22'),
+    ((SELECT id FROM animals WHERE name = 'Gabumon'), (SELECT id FROM vets WHERE name = 'Jack Harkness'), '2021-2-2'),
+    ((SELECT id FROM animals WHERE name = 'Pikachu'), (SELECT id FROM vets WHERE name = 'Maisy Smith'), '2020-1-5'),
+    ((SELECT id FROM animals WHERE name = 'Pikachu'), (SELECT id FROM vets WHERE name = 'Maisy Smith'), '2020-3-8'),
+    ((SELECT id FROM animals WHERE name = 'Pikachu'), (SELECT id FROM vets WHERE name = 'Maisy Smith'), '2020-3-14'),
```
Comment (danmainah) [inline data.sql:316]: - [ ] @MohamedHNoor kindly check your date its suppose to be Pikachu visited Maisy Smith on May 14th, 2020.

File: data.sql [hunk 2]
```diff
@@ -251,4 +251,81 @@ WHERE name IN ('Charmander', 'Squirtle', 'Blossom');
...
+    ((SELECT id FROM animals WHERE name = 'Devimon'), (SELECT id FROM vets WHERE name = 'Stephanie Mendez'), '2021-3-4'),
```
Comment (danmainah) [inline data.sql:317]: @MohamedHNoor kindly check your date its suppose to be Devimon visited Stephanie Mendez on May 4th, 2021.

File: data.sql [hunk 3]
```diff
@@ -251,4 +251,81 @@ WHERE name IN ('Charmander', 'Squirtle', 'Blossom');
...
+    ((SELECT id FROM animals WHERE name = 'Plantmon'), (SELECT id FROM vets WHERE name = 'Maisy Smith'), '2021-4-7'),
+    ((SELECT id FROM animals WHERE name = 'Squirtle'), (SELECT id FROM vets WHERE name = 'Stephanie Mendez'), '2019-9-29'),
+    ((SELECT id FROM animals WHERE name = 'Angemon'), (SELECT id FROM vets WHERE name = 'Jack Harkness'), '2020-10-3'),
+    ((SELECT id FROM animals WHERE name = 'Angemon'), (SELECT id FROM vets WHERE name = 'Jack Harkness'), '2020-11-4'),
+    ((SELECT id FROM animals WHERE name = 'Boarmon'), (SELECT id FROM vets WHERE name = 'Maisy Smith'), '2019-1-24'),
+    ((SELECT id FROM animals WHERE name = 'Boarmon'), (SELECT id FROM vets WHERE name = 'Maisy Smith'), '2019-5-15'),
+    ((SELECT id FROM animals WHERE name = 'Boarmon'), (SELECT id FROM vets WHERE name = 'Maisy Smith'), '2020-2-27'),
+    ((SELECT id FROM animals WHERE name = 'Boarmon'), (SELECT id FROM vets WHERE name = 'Maisy Smith'), '2020-8-3'),
+    ((SELECT id FROM animals WHERE name = 'Blossom'), (SELECT id FROM vets WHERE name = 'Stephanie Mendez'), '2020-8-24'),
```
Comment (danmainah) [inline data.sql:329]: - [ ] @MohamedHNoor kindly check your date its suppose to be Blossom visited Stephanie Mendez on May 24th, 2020.

**Review by CollinsTatang — APPROVED** [review]

**Review by danmainah — CHANGES_REQUESTED** [review]
Hi @MohamedHNoor , 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights
✔️ Gitflow was correctly used
✔️  Good design


# Required Changes ♻️

- [ ] kindly redo the screenshot for the ` List all animals that visited Stephanie Mendez between April 1st and August 30th, 2020.` query, as you had some mistake in data entry that could impact the result of the query.

_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).

**Comment by danmainah** [comment]
- [ ] @MohamedHNoor there seems to be an error with the dashboard, I have informed the technical team, and they will update your dashboard with right status

**Review by EricMbouwe — APPROVED** [review]
Hi @MohamedHNoor, 

Your project is complete! ⚡🔥  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Highlights 🎉🔥
- Everything that creates or change the tables structure should be saved in the schema.sql file. ✔
- Everything that populates tables with data should be saved in the data.sql file. ✔
- Everything that is a query that helps to analyze data ✔


## Optional suggestions ☂

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**